sdk/communication/Azure.Communication.Identity/src/CommunicationIdentityClient.cs

```diff
@@ -318,5 +368,17 @@ public virtual async Task<Response<AccessToken>> GetTokenForTeamsUserAsync(GetTo
                 throw;
             }
         }
+
+        private static int? GetTokenExpirationInMinutesOrNull(TimeSpan tokenExpiresAfter)
```

Comment:
```suggestion
        private static int? GetTokenExpirationInMinutes(TimeSpan tokenExpiresAfter)
```
maybe this way, cause it's clear from return type that it can be nullable

Reply:
agreed

Reply:
Renamed :-)